kiosk/interfaces/hero/index.ts

```diff
@@ -0,0 +1,45 @@
+export interface HeroResult {
+  id: string;
+  advertisement_Title: string;
+  advertisement_Body: string;
+  advertisement_Slogan: string;
+  advertisement_Eyebrow: string;
+  content_Name: string;
+  advertisement_Logo: Image;
+  advertisement_Image: Image;
+  advertisement_Background: Image;
+}
+
+export interface Image {
+  results: AssetResult[];
+}
+
+export interface HeroResults {
```

Comment:
This interface is used only at one place. There is no need to export it and no need to have a separate interface for it. Merge the content of this interface in the only interface using it.